fastseq_cli/transformers_generate.py

```diff
@@ -533,6 +568,11 @@ def run_generate():
                         help="The parameter for repetition penalty. 1.0 means no penalty.")
     parser.add_argument("--do_sample", action="store_true",
                         help="Whether or not to use sampling ; use greedy decoding otherwise.")
+    parser.add_argument("--num_return_sequences", type=int,
+                        default=None, required=False, 
+                        help="The number of independently computed returned sequences for each element in the batch.")
+    parser.add_argument("--seed", type=int, default=None, required=False,
+                        help="Specify a random seed for initialization")
```

Comment:
As the parameters have been changed, do the doc [here](https://github.com/microsoft/fastseq#use-command-line-tool-to-speedup-transformers-models), [here](https://github.com/microsoft/fastseq/tree/main/examples/gpt2#setting), and [here](https://github.com/microsoft/fastseq/tree/main/examples/t5#setting)  need to be changed as well?

Reply:
The necessary change (related to --causal lm) was made in the previous pr. The default behavior is still the same so these examples don't need to change.